docs/BACKUP.md

```diff
@@ -82,6 +82,37 @@ spec:
 ```
 By providing the authentication details and the TLS configuration via references to `Secret` keys, this example will store the backups in a local Minio instance.
 
+Alternatively you can use dynamic credentials from an EKS Service Account using EKS Pod Identity or IRSA:
```

Comment:
Thank you for updating the docs, very much appreciated!.

This documentation is live, we can't document a feature that hasn't been released. I have ported it to our release branch already:
- https://github.com/mariadb-operator/mariadb-operator/commit/febc6f4128ab87d2a87c3c5c900b5faf78b8cdf4

Do you mind undoing this change?